Wikipedia/Code/WMFWelcomeAnalyticsViewController.swift

```diff
@@ -17,29 +36,40 @@ class WMFWelcomeAnalyticsViewController: UIViewController {
             toggle.isOn = false
             BITHockeyManager.shared().crashManager.crashManagerStatus = .alwaysAsk
         }
-        self.view.wmf_configureSubviewsForDynamicType()
+        view.wmf_configureSubviewsForDynamicType()
+        
+        linkLabel.addGestureRecognizer(UITapGestureRecognizer.init(target: self, action: #selector(showPrivacyAlert(_:))))
     }
     
     fileprivate func updateToggleLabelTitleForUsageReportsIsOn(_ isOn: Bool) {
-        
         //Hide accessibility of label because switch will become the label by default.
         toggleLabel.isAccessibilityElement = false
         
-        let title = isOn ? String.localizedStringWithFormat(WMFLocalizedString("welcome-volunteer-thanks", value:"%1$@ Thank you!", comment:"Text which is shown if the user decides to send usage reports. %1$@ will be substituted with a positive image or emoji such as a smiley face or a thumbs-up.\n{{Identical|Thank you}}"), "😀") : WMFLocalizedString("welcome-volunteer-send-usage-reports", value:"Send usage reports", comment:"Text for switch allowing user to choose whether to send usage reports")
-        toggleLabel.text = title
+        toggleLabel.isHidden = !isOn
         toggle.accessibilityLabel = WMFLocalizedString("welcome-volunteer-send-usage-reports", value:"Send usage reports", comment:"Text for switch allowing user to choose whether to send usage reports")
     }
     
     @IBAction func toggleAnalytics(withSender sender: UISwitch){
         if(sender.isOn){
             BITHockeyManager.shared().crashManager.crashManagerStatus = .autoSend
             UserDefaults.wmf_userDefaults().wmf_setSendUsageReports(true)
-
         }else{
             BITHockeyManager.shared().crashManager.crashManagerStatus = .alwaysAsk
             UserDefaults.wmf_userDefaults().wmf_setSendUsageReports(false)
-        
         }
         updateToggleLabelTitleForUsageReportsIsOn(sender.isOn)
     }
+    
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+        linkLabel.font = UIFont.wmf_preferredFontForFontFamily(.systemBold, withTextStyle: .footnote, compatibleWithTraitCollection: traitCollection)
```

Comment:
If `linkLabel` is converted to a button, you can use `linkButton.titleLabel.font`